packages/payments-plugin/README.md

```diff
@@ -1,3 +1,19 @@
 # Payments plugin
 
 For documentation, see https://www.vendure.io/docs/typescript-api/payments-plugin
+
+## Development
+
+### Mollie local development
```

Comment:
Move dev docs to readme instead of public plugin docs, because its only for contributors.

Reply:
Yes this makes sense.